test/WebJobs.Script.Tests/SamplesEndToEndTests.cs

```diff
@@ -617,6 +621,14 @@ private bool IsHostRunning()
 
             public void Dispose()
             {
+                var manager = _config.DependencyResolver.GetService<WebScriptHostManager>();
```

Comment:
I don't think these were a direct contributor to the failing tests but it made the logs a little more confusing -- I kept seeing TimerTrigger-CSharp fire every minute when that host should have been stopped. And also with only 2 cores on the CI machine, it's best to not be running hosts when we don't have to.

I'll leave this in so things are cleaner -- when the Autofac change is (maybe) reverted, lets remove this.

---
In reply to: [77939150](https://github.com/Azure/azure-webjobs-sdk-script/pull/604#discussion_r77939150) [](ancestors = 77939150)